tests/modules/nf-core/smoove/main.nf

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { SMOOVE } from '../../../../modules/nf-core/smoove/main.nf'
+
+workflow test_smoove {
+    
+    bam_tuple_ch = Channel.of([ [ id:'test' ], // meta map
+                               file(params.test_data['homo_sapiens']['illumina']['test_paired_end_sorted_bam'], checkIfExists: true),
+                               file(params.test_data['homo_sapiens']['illumina']['test_paired_end_sorted_bam_bai'], checkIfExists: true),
+                               []
```

Comment:
```suggestion
                               file(params.test_data['homo_sapiens']['illumina']['test_paired_end_sorted_bam_bai'], checkIfExists: true)
```

There is no third input file here :)

Reply:
Changed. I copy/pasted from another module though, unfortunately I don't remember which one...

Reply:
no problem copy/paste errors happen often :stuck_out_tongue: